chitchat/src/delta.rs

```diff
@@ -83,7 +84,6 @@ impl TryFrom<u8> for DeltaOpTag {
         match tag_byte {
             0u8 => Ok(DeltaOpTag::Node),
             1u8 => Ok(DeltaOpTag::KeyValue),
-            2u8 => Ok(DeltaOpTag::NodeToReset),
             _ => {
```

Comment:
You will need to handle 2u8 for SetMaxVersion here.

Reply:
Great point!